.github/workflows/build-workflow.yaml

```diff
@@ -0,0 +1,96 @@
+name: Build and Test
+
+on:
+  # Run when pushing to stable branches
+  push:
+    branches:
+    - 'master'
+    - 'release-*'
+  # Run on branch/tag creation
+  create:
+  # Run on pull requests
+  pull_request:
+
+env:
+  elixir_version: 1.8.1
+  otp_version: 21.3
+
+jobs:
+  test-dialyzer:
+    name: Check Dialyzer
+    runs-on: ubuntu-latest
+    env:
+      MIX_ENV: ci
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/cache@v1
+      with:
+        path: deps
+        key: ${{ runner.os }}-mix-${{ hashFiles(format('{0}{1}', github.workspace, '/mix.lock')) }}
+        restore-keys: |
+          ${{ runner.os }}-mix-${{ hashFiles(format('{0}{1}', github.workspace, '/mix.lock')) }}
+          ${{ runner.os }}-mix-
+    - uses: actions/cache@v1
+      with:
+        path: _build
+        key: ${{ runner.os }}-_build-${{ github.sha }}
+        restore-keys: |
```

Comment:
I think this should be

${{ runner.os }}-_build-${{ github.sha }}
${{ runner.os }}-_build-

same for all other occurrencies